src/main/java/com/hivemq/client/internal/mqtt/MqttWebSocketConfigImplBuilder.java

```diff
@@ -49,13 +51,18 @@
         return self();
     }
 
+    public @NotNull B queryString(final @Nullable String queryString) {
```

Comment:
This method is missing in the `MqttWebSocketConfigBuilderBase` interface

Reply:
Done